### ISSUE TITLE
Publish ES2015 Distribution

### ISSUE THREAD

**Issue by dhardtke** [tjoskar/ng-lazyload-image#363]
Hey there,

I noticed ng-lazyload-image is distributed in the CJS format. Hence, webpack's `ModuleConcatenationPlugin` tells me this:
```typescript
./app/catalog/genex/exercises/math/linear-algebra/matrix/multiplication/matrix-multiplication.module.ngfactory.js + 10 modules 83.1 KiB {55} [built]
    ModuleConcatenation bailout: Cannot concat with ./app/app.constants.ts
    ModuleConcatenation bailout: Cannot concat with ./node_modules/@angular/router/router.ngfactory.js
    ModuleConcatenation bailout: Cannot concat with ./node_modules/@angular/common/fesm2015/common.js
    ModuleConcatenation bailout: Cannot concat with ./node_modules/@angular/common/fesm2015/http.js
    ModuleConcatenation bailout: Cannot concat with ./node_modules/@angular/core/fesm2015/core.js (<- Module is referenced from these modules with unsupported syntax: ./node_modules/ng-lazyload-image/src/lazyload-image.directive.js (referenced with cjs require), ./node_modules/ng-lazyload-image/src/lazyload-image.module.js (referenced with cjs require))
```

Is it possible to switch to ES2015 for tree-shaking?

Update:
Just to clarify: If I exclude ng-lazyload-image from my NgModules my overall app.js bundle decreases from ~800 kB to ~700 kB

**Comment by tjoskar** [comment]
Hi,

Yes, I have been thinking about this. 
Since all* Angular devs are using typescript and a bundler (webpack/rollup/etc) how understand ES modules it should be possible to just publish the ts files and let the user compile them or at least publish es modules files. 
I will fix this in the next release.

**Comment by tjoskar** [comment]
This is fixed in version 5.0.0 🎉
